mappings/net/minecraft/client/render/debug/ChunkBorderDebugRenderer.mapping

```diff
@@ -1,4 +1,6 @@
 CLASS net/minecraft/unmapped/C_anglpvjn net/minecraft/client/render/debug/ChunkBorderDebugRenderer
+	FIELD f_aliphypp CYAN I
```

Comment:
This is `CELL_BORDER` in Mojmap

Reply:
its `0x00FFFF`, so cyan is describing the color